smartrack.rb

```diff
@@ -4,15 +4,16 @@
 
 @xml_urls = []
 @devices = 'IE 11'
-@urls = ['http://www.google.com/'] #initialize URLs array
-
-# while (line = inFile.gets)
-#inFile = File.open('/Users/joshkemp/Ruby_Talk/test_urls.txt', 'r') #read file with site URLs strings
-#while (line = inFile.gets)
-#  @urls << line.to_s.chomp #shove urls from input file into an array
-#end
-#inFile.close
+@urls = []
+login_page = 'https://smartrip.wmata.com/Account/AccountLogin.aspx?'
+
 @driver = Selenium::WebDriver.for :firefox
+@driver.navigate.to login_page
+user_name_field = @driver.find_element(:css, 'input#ctl00_MainContent_txtUsername')
+user_name_field.send_keys 'zr2d2'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.